client/src/components/common/boost/BoostConfigList.stories.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { action } from '@storybook/addon-actions';
-import { addArgs, createStory } from '../utils/stories';
-import BoostConfigList, { BoostConfigListProps } from './BoostConfigList';
+import { addArgs, createStory } from 'utils/stories';
+import BoostConfigList, { BoostConfigListProps } from 'components/common/boost/BoostConfigList';
```

Comment:
Relative path. Seems this is a pattern, I'll either leave you to find the rest of the occurances if this was a mistake or otherwise keen to hear why it was done :)

Reply:
This was definitely a mistake. Must have missed them as I think these files were introduced in a merge from the last PR.